predict.py

```diff
@@ -89,7 +96,7 @@ def predict(
             description="provide debugging output in logs", default=False
         ),
     ) -> ConcatenateIterator[str]:
-        prompt = "User: " + prompt + '\nAssistant: '#Uncomment if you want to use for demo with no chat memory.
+        prompt = PROMPT_TEMPLATE.format(instruction=prompt.strip())
```

Comment:
@daanelson, I think you mentioned somewhere that we should allow users to control this with a input param? I think that's a good idea because, otherwise, users will not be able to collect and pass history across multiple chat interactions.

Reply:
talked with @joehoover about this today, modified the code to have a configurable system prompt.